mimic/rest/loadbalancer_api.py

```diff
@@ -26,20 +27,20 @@
 @implementer(IAPIMock, IPlugin)
 class LoadBalancerApi(object):
     """
-    Rest endpoints for mocked Load balancer api.
+    This class registers the load balancer API in the service catalog.
+    It does NOT implement individual endpoints.
```

Comment:
In a sense, it *does* implement endpoints; as in, this is the plugin which dispatches to the relevant objects that implements endpoints.  I guess "implements" is an unhelpful word here.